### PR TITLE
User show

### PR DESCRIPTION
**Pull request by croixk**
Add a user show page. Add testing for user show page, edge cases - only allow logged in users to access user show page. Otherwise, redirect to dashboard and show a flash message

### CODE REVIEW

File: app/views/dashboard/index.html.erb [hunk 1]
```diff
@@ -1,3 +1,9 @@
+<% if flash.any? %>
```
Comment (tjroeder) [inline app/views/dashboard/index.html.erb:1]: This maybe better in a partial. We can refactor this.

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<br>
+<h5>My account details: </h5>
+<p>Name: <%= session[:auth_hash]["info"]["name"] %></p>
+<p>User ID: <%= session[:auth_hash]["uid"] %></p>
```
Comment (tjroeder) [inline app/views/users/show.html.erb:4]: UID is the Google User ID. I think this needs to be the DB user id.

File: spec/features/profile/show_spec.rb [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    end
+
+    it 'no longer shows profile page after user logs out' do
+      visit '/'
+      click_link 'Login with Google'
+      click_link 'Logout'
+      visit '/profile'
+      expect(current_path).to eq('/')
+      expect(page).to have_content('Please login to see user profile information')
+
```
Comment (tjroeder) [inline spec/features/profile/show_spec.rb:30]: Could you add a test for the redirect after destroying the user.

**Review by tjroeder — APPROVED** [review]
Changes are good. All fixes noted are included in issues to be fixed in later PR.
